taker-frontend/src/components/IncompatibleWarning.tsx

```diff
@@ -0,0 +1,21 @@
+import { Alert, AlertDescription, AlertIcon, AlertTitle, CloseButton, Spacer } from "@chakra-ui/react";
+import * as React from "react";
+
+type IncompatibleWarningProps = {
+    onClose: () => void;
+};
+
+export default function IncompatibleWarning({ onClose }: IncompatibleWarningProps) {
+    return (
+        <Alert status="warning">
+            <AlertIcon />
+            <AlertTitle>Your daemon is incompatible with the connected maker!</AlertTitle>
+            <AlertDescription>
+                Running an incompatible version can result in unexpected behaviour. Ensure compatibility by updating to
+                the latest ItchySats version!
```

Comment:
```suggestion
                the latest ItchySats version
```

no need to shout all the time 😝